src/test/resources/features/convertVideo/video.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Video Controller
+
+  In order to convert a file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+  @functional
+  Scenario: Verify that "/convertVideo" can perform "POST" request converting into MP4
+    When I send a POST request to "/convertVideo" with the following form data
+      | file              | @"template/video/demo.mp4"       |
+      | md5               | 16b258600183d1d5dc0631c82de13000 |
+      | exportFormat      | .mp4                             |
+      | name              | videoDemo                        |
+      | videoCodec        | h264                             |
+      | codec             | mp3                              |
+      | extractThumbnail  | true                             |
+      | extractMetadata   | true                             |
+
```

Comment:
Please consider deleting this empty line.

Reply:
I modified it according your suggestion thanks.